src/quill.mention.js

```diff
@@ -160,11 +163,28 @@ class Mention {
     return this.allowedChars.test(s);
   }
 
-  setMentionListPosition(startIndex) {
+  setMentionListPosition(atSignIndex) {
     const containerPos = this.quill.container.getBoundingClientRect();
-    const indexPos = this.quill.getBounds(startIndex);
-    this.mentionContainer.style.top = `${window.scrollY + containerPos.top + indexPos.bottom}px`;
-    this.mentionContainer.style.left = `${window.scrollX + containerPos.left + indexPos.left}px`;
+    const atSignBounds = this.quill.getBounds(atSignIndex);
+
+    if ((atSignBounds.left + 230) > this.quill.container.offsetWidth) {
```

Comment:
230 is not a good value because it should get the width of the mention list and not a arbitrary value.

Reply:
The only way I could figure out to get the correct width was to use a setTimeout which causes a noticeable flicker on the screen when the list is rendered.